### PR TITLE
Docs: Add F-Droid repository instructions

### PR DESCRIPTION
**Pull request by platschi**
#### Proposed Changes

* Add Info for F-Droid repository with Trustroots mobile app

#### Testing Instructions

* Add repository to your F-Droid installation, search for Trustroots, install app. Verify fingerprints.

Fixes https://github.com/Trustroots/trustroots-expo-mobile/issues/3

### CODE REVIEW

File: docs/Fdroid.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Issue
+
+We don't have resources yet to build a separate "native app" for Android so we're using Expo.io framework instead.
+
+Using Expo seems to block inclusion to the official F-Droid repository. As an alternative, a private repository has been set up to serve F-Droid users with the latest Trustroots Android mobile app.
+
+## Repository details
+
+```html
```
Comment (simison) [inline docs/Fdroid.md]: Maybe just ``` without HTML since there's no HTML to syntax highlight?

Comment (simison) [reply docs/Fdroid.md]: Did in https://github.com/Trustroots/trustroots/pull/1046/commits/ea68a5eb728ceb55d99f466c821244f33d3b7ad5

**Comment by simison** [comment]
@platschi Do you think it would be good to explain that notifications won't work in Fdroid version?

**Comment by platschi** [comment]
> Do you think it would be good to explain that notifications won't work in Fdroid version?

Yes, probably a small list of "Limitations" would be helpful (if any). Notifications do not work because the Play Store version uses GCM to push notifications?

**Comment by simison** [comment]
>Notifications do not work because the Play Store version uses GCM to push notifications?

Notifications _alwys_ require you to push them via operating system manufacturer's servers, like Google/Apple for phones and Google, Mozilla, Apple, Microsoft etc for browsers. Any implementation (GCM/Firebase or others) work the same way. Federated notifications don't exist. :-)

**Pull request merged**